src/cache.ts

```diff
@@ -15,17 +15,17 @@ export default class Cache implements CacheInterface {
     return this.__cache.get(_key)
   }
 
-  set(key: keyInterface, value: any): any {
+  set(key: keyInterface, value: any) {
     const [_key] = this.serializeKey(key)
     this.__cache.set(_key, value)
     this.notify()
   }
 
-  keys() {
+  keys(): string[] {
```

Comment:
this could be inferred since this.__cache is a Map

Reply:
you are right, thanks :):)